skbio/stats/composition.py

```diff
@@ -708,6 +715,10 @@ def ancom(table, grouping,
 
         `"reject"` indicates if feature is significantly different or not.
 
+    pd.DataFrame
+        A table of features and their percentile abundances in each group. If
+        ``percentiles = []``, this will be an empty data frame.
```

Comment:
`percentiles` can be an iterable, I'd change the second sentence to:

"""
If `percentiles` is empty, this will be an empty data frame.
"""